cmd/daytona/config/ssh_file.go

```diff
@@ -261,7 +261,31 @@ func ExportGPGKey(keyID, projectHostname string) error {
 	return importCmd.Run()
 }
 
-func RemoveWorkspaceSshEntries(profileId, workspaceId, projectName string) error {
+func RemoveSshEntry(profileId, workspaceId, projectName string) error {
+	// Construct the Host entry based on provided arguments
+	hostEntry := fmt.Sprintf("Host %s-%s-%s", profileId, workspaceId, projectName)
+
+	// Path to the SSH config file (e.g., ~/.ssh/daytona_config)
+	sshDir := filepath.Join(os.Getenv("HOME"), ".ssh")
+	configPath := filepath.Join(sshDir, "daytona_config")
+
+	// Construct the sed command:
+	// sed -i '/^Host profileId-workspaceId-projectName\s*$/,/^\s*$/d' ~/.ssh/daytona_config
+	sedCommand := fmt.Sprintf(`/^%s\\s*$/,/^\\s*$/d`, hostEntry)
+
+	// Execute the sed command using exec.Command
+	cmd := exec.Command("sed", "-i", sedCommand, configPath)
```

Comment:
This might not work on all OSs. Please use native go (replace with regex and then write to the file with os.WriteFile)